install.ts

```diff
@@ -0,0 +1,81 @@
+import { parse } from "https://deno.land/std@0.136.0/flags/mod.ts";
+import { red } from "https://deno.land/std@0.136.0/fmt/colors.ts";
+
+/*
+ * Prompt for a response
+ */
+async function prompt(message = "") {
+  const buf = new Uint8Array(1024);
+  await Deno.stdout.write(new TextEncoder().encode(message + ": "));
+  const n = <number> await Deno.stdin.read(buf);
+  return new TextDecoder().decode(buf.subarray(0, n)).trim();
+}
+
+export async function checkVersion(version: string): Promise<string> {
+  console.log("Looking up latest version...");
+
+  const versionMetaUrl = "https://cdn.deno.land/strawman/meta/versions.json";
+  const { latest, versions } = await (await fetch(versionMetaUrl)).json();
+
+  if (version === "latest") {
+    version = latest;
+  } else if (!versions.includes(version)) {
+    version = "v" + version;
+    if (!versions.includes(version)) {
+      console.log(`${red("error")}: version(${version}) not found!`);
+
+      const promptForListAvailable = await prompt(
```

Comment:
```suggestion
      const shouldListAvailableVersions = prompt(
```

1. `shouldListAvailableVersions` - for clarity
2. As a follow-up to using the built-in `prompt` function, `await` has to go, because the built-in `prompt` function is synchronous and blocking.